### PR TITLE
Update from fork

### PR DESCRIPTION
**Pull request by vidartf**
This PR brings back in some changes from my fork:
- Adds my info for maintainer in setup.py.
- Adds support for tornado 5.
- Adds support for pytest 4 (and requires pytest >= 3.6).
- Fixes some Python 3 deprecations.

### CODE REVIEW

**Comment by okin** [comment]
Wouldn't it make sense to remove the combinations with old pytest versions from the build matrix if this new version requires at least pytest 3.6 (stated in setup.py)?

Also I'd suggest adding tests for the latest pytest which as of this moment seems to be 4.3.1. 

In this regard I think it would make sense to make use of the [compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release) version statement at least for the pytest part. We don't want to need to update the travis file every time pytest releases a patch version.

**Comment by vidartf** [comment]
> Wouldn't it make sense to remove the combinations with old pytest versions from the build matrix if this new version requires at least pytest 3.6 (stated in setup.py)?

@okin Hah, yes, I missed that. Will fix.

> Also I'd suggest adding tests for the latest pytest which as of this moment seems to be 4.3.1.

Thought that was in, but it was in another branch were I'm trying for tornado 6 compat.

> In this regard I think it would make sense to make use of the compatible release version statement at least for the pytest part. We don't want to need to update the travis file every time pytest releases a patch version.

Are you suggesting we use `4` in the travis config then? Your link seem to be related to version specifiers in `setup.py` so I'm a little confused.

**Comment by vidartf** [comment]
Ah, I figure out what you meant: Use `~=` in the `pip install` line, right?

**Comment by okin** [comment]
@vidartf Yes, I was referring to the usage of `~=` in combination with `pip install`. Your changes look good to me!

File: .travis.yml [hunk 1]
```diff
@@ -5,26 +5,15 @@ python:
     - 3.5
     - 3.6
```
Comment (Jamim) [inline .travis.yml]: I believe that it worth to add `3.7` as well.

Comment (vidartf) [reply .travis.yml]: I agree, but I will do that in a follow-up PR for easier tracking 👍

Comment (vidartf) [reply .travis.yml]: #44.

**Pull request merged**